lib/cli/src/commands/run/wasi.rs

```diff
@@ -483,7 +482,21 @@ impl Wasi {
             mapped_dirs.push(mapping);
         }
 
-        Ok(mapped_dirs)
+        if !have_current_dir {
+            let current_dir =
+                std::env::current_dir().context("could not determine current directory")?;
```

Comment:
This needs to be explicit, we shouldn't do this (basically only `wasmer run XYZ --dir=.` should mount the current dir